sphinx/source/userguide/index.rst

```diff
@@ -5,18 +5,32 @@ Installation
 .. contents:: Table of Contents
```

Comment:
According to your point, shouldn't this file be only generated when the inizialisation is performed?